docs/src/main/sphinx/connector/hive.rst

```diff
@@ -393,6 +387,48 @@ Property Name                                      Description
                                                    is limited to number of buckets.
 ================================================== ============================================================ ============
 
+ORC format configuration properties
+-----------------------------------
+
+The following properties can be used to configure the read and write operations
+with ORC files performed by the Hive connector.
+
+.. list-table:: ORC format configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - ``hive.orc.time-zone``
+      - Sets the default time zone for legacy ORC files that did not declare a
+        time zone.
+      - JVM default
+    * - ``hive.orc.use-columns-names``
+      - Access ORC columns by name. By default, columns in ORC files are
+        accessed by their ordinal position in the Hive table definition. The
+        equivalent catalog session property is ``orc_use_column_names``.
+      - ``false``
+
+Parquet format configuration properties
+---------------------------------------
+
+The following properties can be used to configure the read and write operations
+with Parquet files performed by the Hive connector.
+
+.. list-table:: Parquet format configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - ``hive.parquet.time-zone``
+      - Adjusts timestamp values to a specific time zone. For Hive 3.1+, set
+        this to UTC.
+      - JVM default
+    * - ``hive.parquet.use-columns-names``
+      - Access Parquet columns by name by default. You can change to access
+        columns by their ordinal position in the Hive table definition by
+        setting this property to ``false``. The equivalent catalog session
```

Comment:
Don't use "can" Instead:

Set this property to ``false`` to access columns by their ordinal position in the Hive table definition.

Reply:
I agree with Annie's suggestions, outside of that good to go.